### DISCUSSION TITLE
Incongruenze Tabella 97 Consolati

### DISCUSSION THREAD

**Discussion by LucaBonini64** [italia/ansc#839] [Supporto servizi cooperativi] [answered]
Buongiorno,
inserendo la gestione della tabella Consolati ci siamo accorti che la suddetta Tabella contiene 34 consolati non presenti nelle tabelle ANPR dei consolati, quali AMSTERDAM, BASTIA, LIEGI, LOSANNA, TEGUCIGALPA, ecc.
Altri invece sono doppi: 126 - MONACO PRINCIPATO e 225 - PRINCIPATO DI MONACO.
Vorremmo evitare di gestire una doppia tabella dei consolati ed anche per il comune diventa difficile capire come un consolato possa essere valido per un ufficio e non per l'altro.
Vi chiedo cortesemente se è possibile far coincidere le due tabelle.
Grazie.
Luca Bonini APKAPPA

**Comment by mttfranci** [comment, marked as answer]
Nella tabella di ANSC, vengono mostrati i consolati cessati, ecco il motivo dell'appartente incongruenza.
Ad esempio AMSTERDAM non è più valido dal 2014-06-30 00:00:00.0
Mentre 225 | PRINCIPATO DI MONACO | 1900-01-01 00:00:00.0 | **2007-07-01 00:**00:00.0 dal 2007 (quindi al momento va usato 126 - MONACO PRINCIPATO)
I valori precedentemente validi sono mantenuti come storico per decodificare eventuali dati già inseriti. Se credete potete filtrare le sole righe che non hanno una data fine validità nel passato.

Saluti

Assistenza ANSC
